Goal: Resume properly after ^Z

Fix: #414542

Author: dean gaudet <dean@arctic.org>

Status wrt upstream: should be forwarded

Index: shadow-4.1.0/src/vipw.c
===================================================================
--- shadow-4.1.0.orig/src/vipw.c
+++ shadow-4.1.0/src/vipw.c
@@ -208,7 +208,7 @@
 		pid = waitpid (pid, &status, WUNTRACED);
 		if (WIFSTOPPED (status)) {
 			kill (getpid (), SIGSTOP);
-			kill (getpid (), SIGCONT);
+			kill (pid, SIGCONT);
 		} else
 			break;
 	}
